.changeset/eight-numbers-remember.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystone-next/keystone': patch
+---
+
+Renamed helper function `sessionStrategy` to `asSessionStrategy` to avoid naming clashes.
```

Comment:
Really feel like this shouldn't have a changeset, it doesn't impact the API or behaviour of the package at all.